feature-libs/my-account/organization/src/components/cost-center/form/cost-center-form.service.ts

```diff
@@ -16,7 +21,10 @@ export class CostCenterFormService {
   }
 
   protected build(form: FormGroup) {
-    form.setControl('code', new FormControl('', Validators.required));
+    form.setControl(
+      'code',
+      new FormControl('', [Validators.required, this.codeValidator])
```

Comment:
Thats nice, but what if someone create cost center in other way (eg in backoffice) or cheat in other way? Stan's idea was to have `create` route with dobule slash (//create) and encoding causes, that data route will get other text

Reply:
discussed and decided to remove it